irohad/main/application.cpp

```diff
@@ -45,20 +45,20 @@ using namespace std::chrono_literals;
 // TODO: @muratovv 12.11.2018 remove the mock after effective implementation
 // will be finished IR-1840
 #include <gmock/gmock.h>
-class MockTxPresenceCache : public iroha::ametsuchi::TxPresenceCache {
- public:
-  MOCK_CONST_METHOD1(check,
-                     iroha::ametsuchi::TxCacheStatusType(
-                         const shared_model::crypto::Hash &hash));
-
-  MOCK_CONST_METHOD1(
-      check,
-      iroha::ametsuchi::TxPresenceCache::BatchStatusCollectionType(
-          const shared_model::interface::TransactionBatch &));
-};
-
 namespace iroha {
   namespace ametsuchi {
+    class MockTxPresenceCache : public iroha::ametsuchi::TxPresenceCache {
+     public:
+      MOCK_CONST_METHOD1(check,
+                         iroha::ametsuchi::TxCacheStatusType(
+                             const shared_model::crypto::Hash &hash));
```

Comment:
For the purpose of consistency, name of `hash` argument should be specified here